gravitl/netclient
# Gra 985/proxy reset

Review After PR #139 is merged

Testing Steps: 
1. change proxy port on UI and check if daemon restarts and proxy server starts on updated port
2. check netclient.yml file whether port fields are updated ( `public_listen_port` ,`proxy_listen_port`)
